test/clj/rems/test_themes.clj

```diff
@@ -35,6 +35,6 @@
 (deftest get-theme-attribute-test
   (with-redefs [config/env {:theme {:test "success"
                                     :test-color 2}}]
-    (is (= 2 (styles/get-theme-attribute :test-color)))
-    (is (= "success" (styles/get-theme-attribute :test)))
-    (is (nil? (styles/get-theme-attribute :no-such-attribute)))))
+    (is (= 2 (style-utils/get-theme-attribute :test-color)))
+    (is (= "success" (style-utils/get-theme-attribute :test)))
+    (is (nil? (style-utils/get-theme-attribute :no-such-attribute)))))
```

Comment:
Should these util tests be now in their own file / in the style utils file? Could be in a future PR.

Reply:
This is a synthetic test namespace anyway. I mean, there's no rems.themes namespace that this tests, it tests the concept of themes. So I'm fine with them being here.

Reply:
Yeah works for me. This kind of unit test can now be moved without adding burden to the style file.